lib/vdsm/network/ipwrapper.py

```diff
@@ -19,6 +19,7 @@
 
 from __future__ import absolute_import
 from __future__ import division
+from functools import partial
```

Comment:
This import is from stdlib, it should be in the group below.

Reply:
done